docs/examples/101/api-management-modular/NameValues.bicep

```diff
@@ -0,0 +1,42 @@
+// ****************************************
+// Azure Bicep Module:
+// Create Name / Value pairs from sample JSON records
+// ****************************************
+@minLength(1)
+param apimInstanceName string
+//Name-Value Pair records
+var apimNameValueSet = [
+  {
+    'displayName': 'NameValue1'
+    'value': 'SomeValue1'
+    'tags': [
+      'Example'
+    ]
+    'isSecret': false
+  }
+  {
+    'displayName': 'NameSecretValue'
+    'value': 'SomeSecretValue'
+    'tags': [
+      'Example'
+      'AnotherExampleTag'
+    ]
+    'isSecret': true
+  }
+]
+
+//APIM name value pairs
+resource apiManagementNVPair 'Microsoft.ApiManagement/service/namedValues@2020-06-01-preview' = [for nv in apimNameValueSet: {
+  name: '${apimInstanceName}/${nv.displayName}'

```

Comment:
same feedback on existing here as `groups.bicep`

Reply:
Done and rectified